pages/models.py

```diff
@@ -2586,3 +2587,69 @@ class InstitutionalPartnership(Page):
 
     parent_page_type = ['pages.HomePage']
     max_count = 1
+
+
+class HeroJourneyPage(Page):
+    books = StreamField([
+        ('heading', blocks.CharBlock()),
+        ('subheading', blocks.CharBlock()),
+        ('description', blocks.TextBlock()),
+        ('book_heading', blocks.CharBlock()),
+        ('book_description', blocks.TextBlock()),
+        ('books_link', blocks.CharBlock()),
+        ('books_link_text', blocks.CharBlock()),
+    ], null=True)
+    quiz = StreamField([
+        ('heading', blocks.CharBlock()),
+        ('skip_link', blocks.CharBlock()),
+        ('skip_link_text', blocks.CharBlock()),
+        ('complete_message', blocks.CharBlock()),
+        ('questions', blocks.ListBlock(blocks.StructBlock([
+            ('question', blocks.CharBlock()),
+            ('answers', blocks.ListBlock(blocks.StructBlock([
+                ('text', blocks.CharBlock()),
+                ('correct', blocks.BooleanBlock(required=False))
+            ])
+            ))
+        ])))
+    ], null=True)
+    quiz_complete = StreamField([
+        ('heading', blocks.CharBlock()),
+        ('description', blocks.TextBlock()),
+        ('instructions', blocks.TextBlock()),
+        ('link_url', blocks.CharBlock()),
+        ('link_text', blocks.CharBlock())
+    ], null=True)
+    share = StreamField([
+        ('heading', blocks.CharBlock()),
+        ('description', blocks.TextBlock()),
+        ('instructions', blocks.TextBlock()),
+    ], null=True)
+    thanks = StreamField([
+        ('heading', blocks.CharBlock()),
```

Comment:
Sonya made heading pretty long, here - something like 185 characters, so you may need to change its definition, I don't konw. She also added a `description` (like all the other descriptions), so we might as well add that.

Reply:
I've changed this to a TextBlock and added a description field.